dapp/src/pages/DashboardPage/AcrePointsCard.tsx

```diff
@@ -47,10 +51,19 @@ export default function AcrePointsCard(props: CardProps) {
       py={5}
       {...props}
     >
-      <CardHeader p={0} mb={2}>
+      <CardHeader p={0} mb={2} as={HStack} justify="space-between">
         <TextMd fontWeight="bold" color="grey.700">
           Your Acre points balance
         </TextMd>
+
+        <InfoTooltip
+          label={
+            isWalletConnected
+              ? "Your current balance of Acre points collected so far. New points drop daily and are ready to be claimed. Unclaimed points roll over to the next day."
+              : "Total points distributed to Acre users so far. New points drop daily and can be claimed in each user's dashboard."
```

Comment:
The component does not show the total points when the user is not connected. We cannot add this tooltip variant unless we update the way we display points for non-connected wallets.

Reply:
Good catch. I removed the variant for disconnected wallet.
The tooltip with appropriate message appear as the wallet is connected.
Left TODO comment.
Ref commit: 6e293bdd 8d531a65